plugin/pkg/auth/authenticator/password/passwordfile/passwordfile.go

```diff
@@ -0,0 +1,73 @@
+/*
+Copyright 2015 Google Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package passwordfile
+
+import (
+	"encoding/csv"
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/GoogleCloudPlatform/kubernetes/pkg/auth/user"
+)
+
+type PasswordAuthenticator struct {
+	passwords map[string]*user.DefaultInfo
```

Comment:
Ah, looking at the code again it's because https://github.com/GoogleCloudPlatform/kubernetes/blob/master/pkg/auth/user/user.go doesn't have a password field. I can create a separate struct that implements the Info interface but holds a password.